packages/react/src/components/Forms/Select/tokens.ts

```diff
@@ -22,4 +22,5 @@ export const tokens = (defaults: DefaultTokens) => ({
   "label-color--focus": defaults.theme.colors["primary-600"],
   "multi-pill-background-color": defaults.theme.colors["greyscale-200"],
   "multi-pill-border-radius": "2px",
+  "multi-pill-max-width": "170px",
```

Comment:
What do you think to use relative value ? `68%` seems to be a good take off.
```suggestion
  "multi-pill-max-width": "68%",
```